dashboard/src/pages/hardwareDetails/HardwareDetailsHeaderTable.tsx

```diff
@@ -0,0 +1,188 @@
+import { Tooltip, TooltipContent, TooltipTrigger } from '@/components/Tooltip';
+import { sanitizeTableValue } from '@/components/Table/tableUtils';
+import { PaginationInfo } from '@/components/Table/PaginationInfo';
+// import { IndeterminateCheckbox } from '@/components/Checkbox/IndeterminateCheckbox';
+
+interface IHardwareHeader {
+  treeItems: Trees[];
+}
+
+const columns: ColumnDef<Trees>[] = [
+  // {
+  //   id: 'select',
+  //   header: ({ table }) => (
+  //     <IndeterminateCheckbox
+  //       {...{
+  //         checked: table.getIsAllRowsSelected(),
+  //         indeterminate: table.getIsSomeRowsSelected(),
+  //         onChange: table.getToggleAllRowsSelectedHandler(),
+  //         disabled: true,
+  //       }}
+  //     />
+  //   ),
+  //   cell: ({ row }) => (
+  //     <IndeterminateCheckbox
+  //       {...{
+  //         checked: row.getIsSelected(),
+  //         disabled: !row.getCanSelect(),
+  //         onChange: row.getToggleSelectedHandler(),
+  //       }}
+  //     />
+  //   ),
+  // },
+  {
+    accessorKey: 'treeName',
+    header: ({ column }): JSX.Element =>
+      TableHeader({
+        column: column,
+        sortable: true,
+        intlKey: 'treeTable.tree',
+        intlDefaultMessage: 'Tree',
+      }),
+    cell: ({ row }): JSX.Element => (
+      <Tooltip>
+        <TooltipTrigger>{row.getValue('treeName')}</TooltipTrigger>
+        <TooltipContent>{row.original.gitRepositoryUrl}</TooltipContent>
+      </Tooltip>
+    ),
+  },
+  {
+    accessorKey: 'gitRepositoryBranch',
+    header: ({ column }): JSX.Element =>
+      TableHeader({
+        column: column,
+        sortable: true,
+        intlKey: 'treeTable.branch',
+        intlDefaultMessage: 'Branch',
+      }),
+  },
+  {
+    accessorKey: 'gitCommitName',
+    header: ({ column }): JSX.Element =>
+      TableHeader({
+        column: column,
+        sortable: true,
+        intlKey: 'treeTable.commitTag',
+        intlDefaultMessage: 'Commit/Tag',
+      }),
+    cell: ({ row }): JSX.Element => (
+      <Tooltip>
+        <TooltipTrigger>
+          {sanitizeTableValue(row.getValue('gitCommitName'), false)}
+        </TooltipTrigger>
+        <TooltipContent>{row.original.gitCommitHash}</TooltipContent>
+      </Tooltip>
+    ),
+  },
+];
+
+const sanitizeTreeItems = (treeItems: Record<string, string>[]): Trees[] => {
+  return treeItems.map(tree => ({
+    treeName: tree['tree_name'] ?? '-',
+    gitRepositoryBranch: tree['git_repository_branch'] ?? '-',
+    gitCommitName: tree['git_commit_name'] ?? '-',
+    gitCommitHash: tree['git_commit_hash'],
+    gitRepositoryUrl: tree['git_repository_url'] ?? '-',
+  }));
+};
+
+export function HardwareHeader({ treeItems }: IHardwareHeader): JSX.Element {
+  const [sorting, setSorting] = useState<SortingState>([
+    { id: 'treeName', desc: false },
+  ]);
+  const [pagination, setPagination] = useState<PaginationState>({
+    pageIndex: 0,
+    pageSize: 10,
```

Comment:
let's start with 5 to reduce the scroll size

Reply:
sure, I also changed the possible items per page values, so any table will be able to be seen with 5 items per page now